ERCS/erc-7629.md

```diff
@@ -1,7 +1,7 @@
 ---
 eip: 7629
-title: Unified Token
-description: Unifying ERC-721 and ERC-20 for seamless asset conversion.
+title: ERC-20/721 Unified Token Interface
+description: Unified ERC-20/ERC-721 interface simplifies token management.
```

Comment:
You still don't really say what "unified" means. Does it mean you can convert your ERC-721 into ERC-20 tokens on the fly? Is this defining a subset of functions common to both ERC-721 and ERC-20? You _need_ to give the reader more information here.